### ISSUE TITLE
1.0 doesn't include nextjs 12 support

### ISSUE THREAD

**Issue by MaxLeiter** [hashicorp/next-remote-watch#40]
Sorry if I'm doing something incorrect, but  I'm getting ```
Error: To use middleware you must provide a `hostname` and `port` to the Next.js Server``` on the client using next-remote-watch with  `next@12.1.7-canary.6`.

next-remote-watch 1.0 doesn't seem to have the fix introduced in https://github.com/hashicorp/next-remote-watch/pull/35. Downloading the zip from the github releases, it contains the old
```js
const app = next({ dev: true, dir: program.root || process.cwd() })
```

instead of
https://github.com/kirill-kruchkov/next-remote-watch/blob/140b51c3846d332d7ca70583f5c49361b054fd19/bin/next-remote-watch#L42-L49

**Comment by kirill-kruchkov** [comment]
Hey @MaxLeiter, i think it hasn't been published yet. The 1.0.0 is from a year ago. @BRKalow could you or someone else generate a new version, if that's fine?

**Comment by atomictangerine** [comment]
Hi! This error is still happening. 🙏

**Comment by elliottsj** [comment]
While the npm release is pending, declaring a Git dependency works for me in `package.json`:

```json
"dependencies": {
  "next-remote-watch": "hashicorp/next-remote-watch#53278dac1c44005dc197c2d4e199ac39cffb5be2"
}
```

**Comment by artmsilva** [comment]
Hi, We're going to use @elliottsj's workaround for now until the publishing issues are resolved

**Comment by BRKalow** [comment]
We've just published v2.0.0 (finally!) with the changes needed to support v12, thanks for your patience y'all!
